### PR TITLE
24638 - fix: incorrect name in amalgamation application

### PR DESCRIPTION
**Pull request by hfekete**
*Issue:24638 *https://github.com/bcgov/entity/issues/24638 

*Description of changes:*
fix: incorrect name in amalgamation application

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

File: src/lang/en.json [hunk 1]
```diff
@@ -119,6 +119,7 @@
       "legalTypes": {
         "CC": "BC Community Contribution Company",
         "BEN": "BC Benefit Company",
+        "CBEN": "BC Benefit Company",
```
Comment (severinbeauvais) [inline src/lang/en.json:122]: Also need to add CCC and CUL.

**Comment by hfekete** [comment]
name is showing as Numbered company, existing problem: 
https://business-dashboard-dev.web.app/TZHHWYKW5f?accountid=3040 
current dashboard:
https://dev.business.bcregistry.gov.bc.ca/TZHHWYKW5f?accountid=3040

File: src/lang/en.json (hunk 1 above)
Comment (hfekete) [reply src/lang/en.json:122]: Do you know specific translation that should be added ? Or where I can get them ?

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-dashboard-dev--pr-87-qbfm7ysy.web.app

File: src/lang/en.json (hunk 1 above)
Comment (severinbeauvais) [reply src/lang/en.json:122]: CCC = same as CC
CUL = same as ULC

**Review by severinbeauvais — APPROVED** [review]
Thanks for the sample business. Looks good in preview URL.

**Pull request merged**